lib/matplotlib/axes/_base.py

```diff
@@ -2208,47 +2208,64 @@ def update_datalim_bounds(self, bounds):
         """
         self.dataLim.set(mtransforms.Bbox.union([self.dataLim, bounds]))
 
-    def _process_unit_info(self, xdata=None, ydata=None, kwargs=None):
+    def _process_unit_info(self, datasets=None, kwargs=None, *, convert=True):
         """
-        Look for unit *kwargs* and update the axis instances as necessary
+        Set axis units based on *datasets* and *kwargs*, and optionally apply
+        unit conversions to *datasets*.
 
+        Parameters
+        ----------
+        datasets : list
```

Comment:
You might consider accepting a single tuple as well. This seems to be a common use case and results in a little less cluttered code. But no strong opinion, it's interal anyway.

Reply:
I don't think it really gains us much, and would make the implementation (and documentation) more complex.  Let's say I wouldn't block a PR implementing that, but won't do it here.

Reply:
Fair enough. Even though the added implementation complexity is just
~~~
if isinstance(datasets, tuple):
    datasets = [datasets]
~~~